packages/arg-parser/package.json

```diff
@@ -13,7 +13,8 @@
   "scripts": {
     "test": "mocha -r \"../../scripts/import-expansions.js\" --timeout 60000 -r ts-node/register \"./{src,lib}/**/*.spec.ts\"",
     "test-ci": "node ../../scripts/run-if-package-requested.js npm test",
-    "lint": "eslint --report-unused-disable-directives \"./{src,test}/**/*.{js,ts,tsx}\"",
```

Comment:
This was somehow messing things up in cygwin. Probably best to have these in shared config anway.